doc/languages-frameworks/python.section.md

```diff
@@ -1514,11 +1514,10 @@ Note: There is a boolean value `lib.inNixShell` set to `true` if nix-shell is in
 
 ### Tools {#tools}
 
-Packages inside nixpkgs are written by hand. However many tools exist in
-community to help save time. No tool is preferred at the moment.
-
-- [nixpkgs-pytools](https://github.com/nix-community/nixpkgs-pytools)
-- [poetry2nix](https://github.com/nix-community/poetry2nix)
+Packages inside nixpkgs must use the `buildPythonPackage` or `buildPythonApplication` function,
+because we can only provide security support for non-vendored dependencies.
+However other tools exist in the community to help save time.
+A list of available tools can be found in [awesome-nix](https://github.com/nix-community/awesome-nix#python).
```

Comment:
That makes me uneasy, because it recommends poetry2nix (which we don't want) and machnix (unsupported). Not sure about pip2nix.

At this point, the only tool I can recommend is [nix-init](https://github.com/nix-community/nix-init).

```suggestion

We recommend [nix-init](https://github.com/nix-community/nix-init) for creating new packages, as it already prefetches the source, parses dependencies for some formats and fills out most things in `meta`.
```

Reply:
I agree that describing nix-init is a good idea.
I believe that if more people see the documentation and use nix-init, it will reduce the review process for the following
1. misconfiguration of meta (especially license)
2. inconsistent parameter order
3. copying from other derivations and forgetting to update
4. forgetting parameters like format, pythonImportsCheck, etc.
5. wrong definition of optional-dependencies
6. formatting

Reply:
fine for me